gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -3791,6 +3811,74 @@ components:
                     example: iOS
                 primaryOwner:
                     $ref: "#/components/schemas/PrimaryOwner"
+        BaseGroup:
+            type: object
+            properties:
+                id:
+                    type: string
+                    description: Group's uuid.
+                    example: 00f8c9e7-78fc-4907-b8c9-e778fc790750
+                name:
+                    type: string
+                    description: Group's name.
+                    example: Gravitee group
+                    minLength: 1
+        Group:
+            type: object
+            allOf:
+                - $ref: "#/components/schemas/BaseGroup"
+                - properties:
+                    eventRules:
+                        type: array
+                        items:
+                            $ref: "#/components/schemas/GroupEvent"
+                        description: Event rules for the group.
+                    manageable:
+                        type: boolean
+                        description: Current user can manage group.
+                    createdAt:
+                        type: string
+                        format: date-time
+                        description: Group's creation date.
+                        example: 2018-11-23T22:00:00.000Z
+                    updatedAt:
+                        type: string
+                        format: date-time
+                        description: Group's last updated date.
+                        example: 2018-11-23T22:00:00.000Z
+                    maxInvitation:
+                        type: integer
+                        description: Maximum number of group members.
+                    apiRole:
+                        type: string
+                        description: The Api role of the group.
+                    lockApiRole:
+                        type: boolean
+                        description: Api role cannot be changed.
```

Comment:
```suggestion
                        description: Api role cannot be changed by a group admin.
```

Reply:
Can it be changed by someone other than a group admin?

Reply:
Just tried to follow the rabbit down the rabbit hole... It doesn't seem very obvious that only admins can change the role or that other users can change the role who are not admin. I'm just going to keep the descriptions as is for now.